.circleci/config.yml

```diff
@@ -47,6 +40,12 @@ jobs:
                      --client-id=50 \
                      --token=$STITCH_API_TOKEN \
                      tests
+      - run:
+          name: 'pylint tests'
+          command: |
+            source /usr/local/share/virtualenvs/tap-tester/bin/activate
+            pip install pylint
```

Comment:
Same comments as other PR, use the same virtual env for pylint test as pylint the code, no need to pip install pylint, no need to pin pip and setuptools versions.

Reply:
Will remove the pins, I do think I want to do the pylint this way to avoid import errors though. I don't think pylint is a req for tap-tester (I might be wrong there).